test/images/agnhost/README.md

```diff
@@ -72,6 +72,121 @@ created with the `pause` argument instead, allowing us execute multiple commands
     kubectl exec test-agnhost -- /agnhost dns-server-list
 ```
 
+The `agnhost` binary is a CLI with the following subcommands:
+
+### net
+
+The goal of this Go project is to consolidate all low-level
+network testing "daemons" into one place. In network testing we
+frequently have need of simple daemons (common/Runner) that perform
+some "trivial" set of actions on a socket.
+
+Usage:
+
+* A package for each general area that is being tested, for example
+  `nat/` will contain Runners that test various NAT features.
+* Every runner should be registered via `main.go:makeRunnerMap()`.
+* Runners receive a JSON options structure as to their configuration. `Run()`
+  should return the disposition of the test.
+
+Runners can be executed into two different ways, either through the
+command-line or via an HTTP request:
+
+Command-line:
+
+```console
+    kubectl exec test-agnhost -- /agnhost net -runner <runner> -options <json>
```

Comment:
I think running commands like this will actually cause problems with cobra:

Cobra uses pflag which enforces the idiom that a single hyphen indicates a "short flag" of one character and double hyphen indicates a multi-char flag.

Each of the commands seems like it would need this doc slightly updated to use the double hyphen and the args for each command changed to 0

Reply:
True. This part of the doc was copied over from the net's README, but yeah, it should be --runer and --options instead.